edk2toolext/environment/extdeptypes/nuget_dependency.py

```diff
@@ -105,13 +105,16 @@ def _fetch_from_cache(self, package_name):
                 return_buffer.seek(0)
                 return_string = return_buffer.read()
                 NugetDependency.global_cache_path = return_string.strip().strip("global-packages: ")
-
+        
+        if NugetDependency.global_cache_path is None:
+            logging.info("Could not determine Nuget global packages cache location")
```

Comment:
should we add return_string into the log so we can evaluate the issue

Reply:
In theory it should already be in the ci buildlog